### ISSUE TITLE
GG Classic - specular camera setting doesn't work

### ISSUE THREAD

**Issue by Ghost-l0l** [TheGameCreators/GameGuruRepo#5686]
In Files/effectbank/reloaded there is the settings.fx file that lets us change some options and shader properities; 
I used it to modify many things, but it seems that the specular camera settings just don't work at all. 
Doesn't matter what values I try, it doesn't change anything. And I know what it should look like, because it worked years ago on previous updates. 

If that's any helpful I can upload an older shader (apbr_core.fx from 2018) that has that specular camera working) 

![image](https://github.com/user-attachments/assets/672be6bb-b556-4edb-9c0c-7c40cc780eb2)

**Comment by synchromesh62** [comment]
Didn't you have to update the blob files \ refresh the shader after you edited them ?
Something like forceloadtestgameshaders = 2 in setup.ini before running your level in the editor.

**Comment by Ghost-l0l** [comment]
Yes yes, every time after you make any changes in settings.fx you must set forceloadtestgameshaders = 1 to reload all shaders, but this specular thing really doesn't work, I was changing many things in settings.fx  and this is the only one that doesn't work.

**Comment by LeeBamberTGC** [comment]
@Ghost-l0l Please do send the apbr_core.fx from 2018 and I will do a WINDIFF. From the way the two defines are commented out, I am guessing this code was superceded with something else a long time ago. I will change to a feature request and find out what these defines do differently than the present handling of specular in apbr_core.fx

**Comment by Ghost-l0l** [comment]
@LeeBamberTGC 
[apbr_core.zip](https://github.com/user-attachments/files/16931466/apbr_core.zip)
That's the apbr_core.fx I was talking about; 

I also packed in zip all shaders from reloaded folder, so you can see all changes I did in settings.fx too:
[reloaded.zip](https://github.com/user-attachments/files/16931510/reloaded.zip)

Thank you for answering! :)

**Comment by Ghost-l0l** [comment]
@LeeBamberTGC 
I would also like to ask to look at this one issue that I reported few weeks ago, but it wasn't tagged in any way, so very possibly went under the radar; I think I already narrowed down where the problem is: https://github.com/TheGameCreators/GameGuruRepo/issues/5464#issuecomment-2285421190
